lib/src/components/_stacks-button-groups.less

```diff
@@ -17,27 +17,27 @@
 
     .s-btn {
         margin-right: 0; // With Safari in production, we need to collapse white space
-
-        // We don't want border-radii on interior buttons
-        &:not(:first-of-type):not(:last-of-type) {
-            border-radius: 0;
-        }
+        border-radius: 0; // Kill the default border radius
```

Comment:
I switched to removing the border radius by default here, and then re-adding it as needed to the first and last children, but I may switch back to the original approach of removing border so `s-btn-group` can actually support different button sizes and their different border radiuses. 🤔